tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -109,3 +113,38 @@ Verify Quick Starts Work As Expected When At Least One Step Is Skipped
     END
     Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}      In Progress
     Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+
+Verify Quick Starts Work As Expected When Restarted And Left In Between
+    [Arguments]     ${element}
+    QuickStart Status Should Be    ${element}  Complete
```

Comment:
after this line, I think it makes sense to add `Run Keyword And Continue On Failure Link Text On QuickStart Card Should Be element=${element} exp_link_text=Restart`